.github/pull_request_template.md

```diff
@@ -33,6 +33,7 @@ We're here to help! This is simply a reminder of what we are going to look for b
 - [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] I have added any necessary documentation (if appropriate)
+- [ ] I have updated `e2e-test-audit.csv` (if a new command or e2e test has been added)
```

Comment:
[np] It's not a csv anymore, as you're using markdown table.

Reply:
I would also link to the
```suggestion
- [ ] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
```

Reply:
One last comment I have on the checklist is there is nothing to force the user to update the table. I see we have a code health check. In a follow-up check-in, I think it would be good to add a check that the table has been updated if a new test has been added, otherwise I can see it falling out of sync very easily.

Reply:
yes, you're right. What I would like to do in a follow-up PR is to automatically update this file. If I cannot achieve it I will add a check.

Reply:
sgtm 👍